spec/client_spec.rb

```diff
@@ -177,12 +177,12 @@ def wait_global_key_missing(key)
 
   it "should tell if index exists" do
     @index.add_object!({ :name => "John Doe", :email => "john@doe.org" }, "1")
-    expect(@index.exists).to be true
+    expect(@index.exists?).to be true
```

Comment:
Do we want to test both since both are valid?

Reply:
I thought that only `exists?` would need to be tested since this is the recommended usage. what do you think?

Reply:
Up to your team's good practices, how do you do in other clients?